forgettable-frontend/src/pages/encounters/CreateEncounter.module.css

```diff
@@ -0,0 +1,65 @@
+@import '../../colours.css';
+
+.Card{
+    display:flex;
+    flex-direction: column;
+    width:910px;
+    height:875px;
+    margin: auto;
+    margin-top: 50px;
+}
```

Comment:
Can we use the saved var values in the index css? like var(--text-xxlarge) etc ?